channeldb/payments.go

```diff
@@ -104,6 +104,14 @@ var (
 	paymentsIndexBucket = []byte("payments-index-bucket")
 )
 
+var (
+	errNoSequenceNumber        = errors.New("sequence number not found")
```

Comment:
in the next commit it looks like it's possible for these errors to be returned via the public interface? if so we should consider exposing them as well and adding godocs